avm/res/document-db/database-account/sql-role/sql-role-definitions/main.bicep

```diff
@@ -0,0 +1,53 @@
+metadata name = 'DocumentDB Database Account SQL Role Definitions.'
+metadata description = 'This module deploys a SQL Role Definision in a CosmosDB Account.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Database Account. Required if the template is used in a standalone deployment.')
+param databaseAccountName string
+
+@description('Optional. An array of data actions that are allowed.')
+param dataActions array = [
+  'Microsoft.DocumentDB/databaseAccounts/readMetadata'
```

Comment:
Can we decide whether these default values are here or in the parent "sql-role/main.bicep" file? These default values are duplicated these two files.